content/community/mailingLists.md

```diff
@@ -50,9 +48,16 @@ decisions, and vote on technical issues.
 * Subscribe: <dev-subscribe@hive.apache.org>
 * Post: <dev@hive.apache.org>
 * Unsubscribe: <dev-unsubscribe@hive.apache.org>
-* Archives:
-    * [Apache][dev_apache]
-    * [Markmail][dev_markmail]
+* Archives: [Apache][dev_apache]
+
+## Issues Mailing List
+---
+The issues list receives notifications about bugs and new features on [JIRA].
```

Comment:
"The issues list receives all notifications from the [JIRA] issue tracker."

Reply:
Done.Thx!